Add component layout overrides (#910)
bitmovin/bitmovin-player-ui: CHANGELOG.md

### Added

- `UIConfig.enableResumeFromLastPosition` option to enable storing playback progress and resume known sources from the last saved position.

> - `UIConfig.componentLayoutOverrides` and `UIComponentLayoutOverride` to include or exclude default UI components without rebuilding complete UI layouts through the `UIFactory`.

Reviewer: nit: I think the current phrasing undersells the feature a bit, I'd emphasize more what it solves 

```suggestion
- `UIConfig.componentLayoutOverrides` and `UIComponentLayoutOverride` to keep or remove supported components from default UI layouts without rebuilding complete `UIFactory` layouts.
```

Author: [ee3212c](https://github.com/bitmovin/bitmovin-player-ui/pull/910/commits/ee3212c36940ef5d82791bcb3310027cee047850)
